utils/defs.h

```diff
@@ -142,11 +145,15 @@ struct config_options
         uint8_t bin_masks[4];
     };
 
+    uint8_t _pad[4];
+    double boxsize_y;
+    double boxsize_z;
```

Comment:
Hmm - why not move these three fields to right below the `union` for boxsize. Saves any padding bytes

Reply:
This way preserves the ABI, but I agree it would be cleaner to keep these vars together. I don't have much preference either way.

Reply:
In that case, let's move the boxsizes up and declared all together